sdk/monitor/tests.yml

```diff
@@ -12,19 +12,8 @@ stages:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
         AZURE_TEST_RUN_LIVE: 'true'
-      Matrix:
-        Linux_Python35:
-          Pool: azsdk-pool-mms-ubuntu-1804-general
-          OSVmImage: MMSUbuntu18.04
-          PythonVersion: '3.5'
-          CoverageArg: '--disablecov'
-        MacOs_Python37:
-          Pool: Azure Pipelines
-          OSVmImage: 'macOS-10.15'
-          PythonVersion: '3.7'
-          CoverageArg: '--disablecov'
-        Linux_Python39:
-          Pool: azsdk-pool-mms-ubuntu-1804-general
-          OSVmImage: MMSUbuntu18.04
-          PythonVersion: '3.9'
-          CoverageArg: ''
+      MatrixConfigs:
+        - Name: Monitor_live_test
+          Path: sdk/monitor/platform-matrix.json
```

Comment:
Do we know if there is a good reason the monitor has a unique matrix?

Reply:
The matrix they set up is slightly different in how the python versions match up against the operating systems. I was aiming for full parity on a first pass. We could import from the base matrix with filters/excludes/overrides, but it wouldn't be exactly the same. Might be worth a conversation about what their matrix needs are and whether we can get them more in line with the base.

Reply:
@rakshith91 gave some input on this, so I'll update the matrix to only exclude 2.7 versions. I'm still thinking about how to get this from the base matrix without excluding macos. This might a reason in favor of adding a mode for "import and combine sparse" in addition to the existing "import and combine full."

Reply:
@benbp On a second thought, do add python 2.7 as well to the test matrix - i'll create a separate PR that excludes only monitor exporter and not other monitor related packages from running 2.7

Reply:
@rakshith91 ok great. In that case I'll just have it run the base matrix then I think? That would be:

![image](https://user-images.githubusercontent.com/1020379/107679248-50a11800-6c6a-11eb-8d11-58db97f59ca2.png)